app/models/admin/oral_history_access_request.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::OralHistoryAccessRequest < ApplicationRecord
+  encrypts :patron_name, :patron_email, :patron_institution, :intended_use
+  belongs_to :work
+  validate :work_exists
+  def work_exists
+    Work.exists?(self.work_id);
+  end
```

Comment:
I can't find this `work_exists` method used anywhere in the code?  

I don't think it's needed.   If the work doesn't exist, `request.work` will be nil. So you can always just check `request.work.present?` whenever you want to check if it exists, no need for this method. (If the method WAS going to remain, it should be end in a `?`, convention for boolean returning methods!

Reply:
I'm using the method for validation: `validate :work_exists`, but this is also enforced at the DB level. Will change the method name -- yes, question mark for boolean return values. I was just skeeved by the idea of saving requests for works that didn't exist, I gues.

Reply:
Hmm, I see. I'm not sure I've ever seen this done before in Rails. i guess I see what you're doing, but it should be extremley unusual for this to happen, the app wouldn't normally allow it, unless I guess a work was deleted right as someone were filling out a request? And then the db will enforce it anyway, so....

I'm just thinking the less code, the less code to maintain/bugs/etc. But if you feel strongly, you could leave the validation I guess.